docs/querying/basics.md

```diff
@@ -109,8 +109,17 @@ single sample value for each at a given timestamp (point in time).  In the simpl
 form, only a metric name is specified, which results in an instant vector
 containing elements for all time series that have this metric name.
 
+The value returned will be that of the most recent sample prior to the query's
+evaluation timestamp (in the case of an
+[instant query](api.md#instant-queries))
+or of the current resolution step within the query (in the case of a
```

Comment:
"resolution step" sounds weird to me. Also, the "of" seems off (no pun intended).

Maybe
```suggestion
or prior to the current step within the query (in the case of a
```